setup.py

```diff
@@ -86,7 +86,8 @@ def generate_proto(source):
         # Cloud TPU VM jaxlib can be installed via:
         # $ pip install jax[tpu] -f https://storage.googleapis.com/jax-releases/libtpu_releases.html
         'tpu': [f'jaxlib=={_current_jaxlib_version}',
-                f'libtpu-nightly=={_libtpu_version}'],
+                f'libtpu-nightly=={_libtpu_version}',
+                'requests'],
```

Comment:
Maybe comment this is needed for jax.distributed.initialize? Or not since it might become stale, up 2 u

Reply:
Good idea, Done